solution-base/build.sh

```diff
@@ -0,0 +1,205 @@
+#!/bin/bash
+set -eu
+
+PWD=$(pwd)
+BUILD_ROOT=${PWD}/_build
+ISO_ROOT=${BUILD_ROOT}/root
+IMAGES_ROOT=${ISO_ROOT}/images
+DEPLOY_ROOT=${ISO_ROOT}/operator/deploy
+
+PRODUCT_NAME=Zenko-Base
+PRODUCT_LOWERNAME=zenko-base
+BUILD_TIMESTAMP=$(date -u +"%Y-%m-%dT%H:%M:%SZ")
+BUILD_HOST=$(hostname)
+
+VERSION_SHORT=$(git describe --abbrev=0)
+VERSION_FULL=${VERSION_SHORT}-dev
+GIT_REVISION=$(git describe --long --always --tags --dirty)
+ISO=${BUILD_ROOT}/${PRODUCT_LOWERNAME}-${VERSION_FULL}.iso
+
+DOCKER=docker
+DOCKER_OPTS=
+DOCKER_SOCKET=${DOCKER_SOCKET:-unix:///var/run/docker.sock}
+HARDLINK=hardlink
+SKOPEO=skopeo
+SKOPEO_OPTS="--override-os linux --insecure-policy"
+OPERATOR_TAG=$(grep /operator: deps.txt | awk -F ':' '{print $2}')
+SOLUTION_REGISTRY=metalk8s-registry-from-config.invalid/${PRODUCT_LOWERNAME}-${VERSION_FULL}
+
+ONESSL_VERSION=v0.13.1
+
+export KUBEDB_SCRIPT_LOCATION="curl -fsSL https://raw.githubusercontent.com/kubedb/installer/v0.13.0-rc.0/"
+export KUBEDB_NAMESPACE=kube-system
```

Comment:
Kubedb defaults to using the kube-system namespace, I don't think we can specify the solution environment here as that is an unknown value. But it may not be ideal to use kube-system in this way.

Reply:
we probably shouldn't deploy in  the `kube-system` namespace anyways.
Since we're going to manually run `kubectl apply`, this can probably be left as a placeholder until then

Reply:
hmm that sounds reasonable, does kubectl have any sort of placeholder or env var completion built in? Or just edit the file? I haven't seen anything about it before.

Reply:
I don't believe the `kubectl` has any placeholders (without bringing in a different tool like `kustomize`).
I was thinking of using `envsubst` or remove the `namespace` line to allow for `kubectl apply` with the `--namespace` option

Reply:
I did some searching after I asked and didn't find anything kubectl related, I replaced it with "SOLUTION_ENV" for now which should be an easy replace. We can decide how later on as we might need to replace more than one thing in the end.